src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,120 @@
+const { expensesService, usersService } = require('./../services');
+const { expenseSchema } = require('./../schemas');
+const joi = require('joi');
+
+const getAll = (req, res) => {
+  const { userId, categories, from, to } = req.query;
+  const expenses = expensesService.getAll({
+    userId: +userId,
+    categories,
```

Comment:
If `categories` is intended to be an array, consider using `categories.includes(expense.category)` to filter expenses by multiple categories.